chef_master/source/install_chef_automate.rst

```diff
@@ -13,24 +13,27 @@ Install Chef Automate
 
 A Chef Automate installation consists of a minimum of two nodes:
 
-* Chef server with push jobs server installed
+* Chef server
 
-  * Contains the cookbooks and data used to build, test, and deploy your components within Chef Automate and your infrastructure.
+  Contains the cookbooks and data used to build, test, and deploy your components within Chef Automate and your infrastructure
 
-  * Runs push jobs, which is used in conjunction with a push jobs client on build nodes to coordinate builds across those nodes.
+* Chef Automate server
 
-   .. note:: If you have an existing Chef server installation, it is best to
-    have a separate Chef organization for managing Chef Automate.
-    Instructions for this can be found below under the heading "Create a User and Organization to Manage Your Cluster".
-    If you don't, you can install a minimal one for use with Chef Automate. See "Chef Server Installation and Configuration" below for details.
+  Coordinates the process of moving a change through the workflow pipeline as well as providing insights and visualizations about your Chef Automate cluster
 
-* Chef Automate server
+Optional components include:
+
+* Push jobs server
+
+  Runs push jobs, which is used to create infrastructure nodes for deployment testing and is also needed if you use push jobs-based build nodes as part of your testing and deployment process
```

Comment:
Should we mention that this is also used by the build nodes. Something that let the user know that if they chose to use build nodes instead of runners they need this.